.github/workflows/ci-lint/action.yml

```diff
@@ -7,6 +7,14 @@ runs:
       run: |
         cargo fmt -- --check
         deno fmt --check ./core/ ./ops/ ./serde_v8/
+    - name: tsc
+      shell: bash
+      run: |
+        # If npm is installed, run tsc
+        if command -v npm &> /dev/null; then
+          npm install -g typescript
```

Comment:
I'm not super happy about it, but that improves our lint checks then fine :)

Reply:
Oh well, bootstrapping is hard

Reply:
Switched this out for deno :D